Makefile

```diff
@@ -72,8 +72,6 @@ install:
 	@# speedup for end-of-life builds
 	if $(PYTHON) -V | fgrep -e 3.5 -e 3.6; then $(PIP) install --prefer-binary opencv-python-headless numpy; fi
 	for mod in $(BUILD_ORDER);do (cd $$mod ; $(PIP_INSTALL) .);done
-	@# workaround for shapely#1598
-	$(PIP) install --no-binary shapely --force-reinstall shapely
```

Comment:
I'm not so sure actually. Remember, this was an [issue](https://github.com/shapely/shapely/issues/1598) of interference between Shapely and Pytorch. We now use a different version on either side. I did not see the problem surface, but should actually check directly. (Unfortunately, this can only be done once I am finished with configuring a GPU-enabled ocrd/all of managable size.)

Reply:
Unfortunately, the problem does persist with Shapely 2.x. I see no other option but to import torch early / shapely late everywhere...

But because ocrd_validators already imports shapely, and ocrd_cli_wrap_processor imports ocrd_validators, this is quite difficult: we would have to import torch all over the place (even in modules which don't need it)...

Reply:
> I see no other option but to import torch early / shapely late everywhere...

Or – of course – to keep compiling shapely via `pip install --no-binary shapely --force-reinstall shapely`.

BTW, currently Kraken also interferes:
```
kraken 4.3.9 requires numpy~=1.23.0, but you have numpy 1.24.2 which is incompatible.
kraken 4.3.9 requires shapely==1.8.5, but you have shapely 2.0.1 which is incompatible.
```

Reply:
> And it looks like [pip configuration](https://pip.pypa.io/en/stable/topics/configuration/) does **not** allow setting `no-binary` statically.

Ah, it [is supported now](https://github.com/pypa/pip/commit/530464b15762907d011f6d0fa006995f91c5961b), just not documented. It does work with a simple `echo no-binary = shapely >> /etc/pip.conf`. We just have to be careful if there already is a config / global section / no-binary line.

Reply:
I am wondering what to do about this problem in native installations with CUDA available though. We cannot just write to /etc/pip.conf or even ~/.config/pip/pip.conf. But we also cannot just sit around and wait for the next `pip install` to bring us the shapely of death. If only there was a way to confine this constraint to the current venv...

Reply:
Well, I was surprised to learn that, unfortunately, no: `pip config set prefer-binary` does not actually seem to work. See the CI failures in the changes leading up to b3618a9.

AFAICT the problem is that pip does not have a correct (workable) syntax for a list of kw options (`shapely=1 opencv-python-headless=1`).